### DISCUSSION TITLE
Consider adding an option to disable Markdown rendering in the input box?

### DISCUSSION THREAD

**Discussion by CodeZhangBorui** [open-webui/open-webui#7403] [General]
I hope an option can be added to not render/detect syntax Markdown in the input box, but to render it again in the chat interface after sending. (Similar to the current behavior of editing messages?)

I wish I could write Markdown syntax myself, but every time I paste my Markdown code into the input box below, Open WebUI tries to modify the syntax/reverse the meaning, resulting in lost line breaks/misaligned formatting. Even manually entering **\`\`\`** is escaped as **\\\`\\\`\\\`**.

Are user settings considered to be added, or are they already provided by Open WebUI?

Open WebUI Version: `ghcr.io/open-webui/open-webui:main` - `9e5f110d9f20` - [v0.4.4](https://github.com/open-webui/open-webui/releases/tag/v0.4.4)

**Comment by tjbck** [comment]
This issue is resolved with the latest!

With that being said, you can also disable the rich input text from settings > interface. Hope that helps!

**Reply by CodeZhangBorui** [reply]
Thanks! That absolutely works well!
